raisinbread/README.md

```diff
@@ -9,7 +9,19 @@ adjusted when existing features are changed or removed.
 The sections below provide some insights into installing, modifying and exporting 
 the dataset.
 
-### Installing RB
+### Automated RB Installation
+The script `tools/raisinbread_refresh.php` includes functionality that drops all existing tables in the
+database and sources all of the RaisinBread data automatically. The script will preserve database settings such as API Keys and server configuration settings to simplify the process of switching between development environments. This tool can be run through the provided Makefile. To run this script, navigate to the LORIS
```

Comment:
```suggestion
database and sources all of the RaisinBread data automatically. The script will preserve some server configuration settings in your database to simplify the process of switching between development environments. This tool can be run through the provided Makefile. To run this script, navigate to the LORIS
```

Hey I made a mistake, API keys are not actually preserved at this time. (https://github.com/aces/Loris/issues/6374).